app/controllers/api/v1/partner_companies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::PartnerCompaniesController < Api::V1::ApiController
+  before_action :cpf_validation, only: %i[search]
+  def search
+    @partner_company_employee = PartnerCompanyEmployee.find_by(cpf: params[:q])
+    if @partner_company_employee.blank?
+      render json: ['Nenhum desconto para esse CPF'], status: :not_found
```

Comment:
```suggestion
      render json: 'Nenhum desconto para esse CPF', status: :not_found
```

Só a mensagem não dá certo?

Reply:
MInha recomendação já seria usar um `hash` ao invés de um `array`. Algo como `render json: { message: 'Nenhum desconto para esse CPF' }`

Reply:
Está ótimo. Arrumamos aqui.